src/v2/Apps/_About2/AboutApp.tsx

```diff
@@ -18,235 +27,406 @@ export const AboutApp: React.FC = () => {
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fit&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FoOhfzetWDyPIdRYLVRBy8A%2Flarge.jpg&width=890&height=1046&quality=80",
+              name: "Caleb Hahne",
+              artworkTitle: "Carry Me Home, 2019",
+              gallery: "1969 Gallery",
+              width: 340,
+              height: 500,
+            },
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fit&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FSbbu7kX5gzxEUfL4Jwyk6Q%2Flarge.jpg&width=890&height=1112&quality=80",
+              name: "Amy Sherald",
+              artworkTitle: "The Make Believer (Monet's Garden), 2016",
+              gallery: "The Studio Museum in Harlem",
+              width: 300,
+              height: 375,
+            },
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fill&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FE_tdYYu4XOeuVK8ZHFwJSw%2Flarge.jpg&width=890&height=890&quality=80",
+              name: "Frank Stella",
+              artworkTitle:
+                "Sinjerli Variations Squared with Colored Grounds VI, 1981",
+              gallery: "Heritage Auctions",
+              width: 300,
+              height: 300,
+            },
+            {
+              imgUrl:
+                "https://d7hftxdivxxvm.cloudfront.net/?resize_to=fit&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2Fm4kmYLBKSkExMXlxjQ4i8A%2Flarge.jpg&width=890&height=1046&quality=80",
+              name: "Inka Essenhigh",
+              artworkTitle: "Old Trees in Fall, 2018",
+              gallery: "Miles McEnery Gallery",
+              width: 340,
+              height: 500,
+            },
+          ]
+
+          return items.map(item => {
+            return <RailComponent {...item} />
+          })
+        }}
+      />
+
+      <SellWithArtsySection />
+      <Spacer mt={4} />
+      <AppDownloadSection />
+      <CollectorInfoSection />
+      <ArtsySpecialistsSection />
+    </>
+  )
+}
+
+interface RailComponentProps {
+  imgUrl: string
+  name: string
+  artworkTitle: string
+  gallery: string
+  width: number
+  height: number
+}
+
+const RailComponent: React.FC<RailComponentProps> = props => {
+  return (
+    <Box>
+      <Image
+        width={props.width}
+        height={props.height}
+        src={props.imgUrl}
+        alt=""
+      />
+      <Text variant="md">{props.name}</Text>
+      <Text variant="md" color="black60">
+        {props.artworkTitle}
+      </Text>
+      <Text variant="xs" color="black60">
+        {props.gallery}
+      </Text>
+    </Box>
+  )
+}
+
+const SellWithArtsySection: React.FC = () => {
+  const image = resized(
+    "http://files.artsy.net/images/molly_green_original.jpeg",
+    { width: 640 }
+  )
+
+  return (
+    <Container bg="black100" color="white100" mt={100}>
```

Comment:
As a good habit, we should avoid putting padding or margin on the inner containers of components themselves and instead move that a bit higher up. Meaning:

Avoid: 
```tsx
const Foo = () => {
  return <Box mb={1}>Hi</Box>
}
const Bar = () => {
  return <Box mb={2}>Hello</Box>
}
const Baz = () => {
  return <Box>There</Box>
}
const App = () => {
  return (
    <>
      <Foo />
      <Bar />
      <Baz />
    </>
  )
}
```

Better: 
```tsx
const Foo = () => {
  return <Box>Hi</Box>
}
const Bar = () => {
  return <Box>Hello</Box>
}
const Baz = () => {
  return <Box>There</Box>
}
const App = () => {
  return (
    <>
      <Foo />
      <Spacer mb={2} />
      <Bar />
       <Spacer mb={2} />
      <Baz />
    </>
  )
}
```

The latter example is a little less clean code-wise, but its more maintainable in the long run.